test/benchmark_test.cc

```diff
@@ -150,7 +150,7 @@ static void BM_LongTest(benchmark::State& state) {
 BENCHMARK(BM_LongTest)->Range(1 << 16, 1 << 28);
 
 static void BM_ParallelMemset(benchmark::State& state) {
-  int size = state.range(0) / sizeof(int);
+  int size = static_cast<int>(state.range(0) / sizeof(int));
```

Comment:
I think it would be more correct to do `st.range(0) / static_cast<int>(sizeof(int))`